src/service/cli/generate.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const fs = require(`fs`);
+const {shuffle, getRandomInt} = require(`../utils/_utils`);
+
+const DEFAULT_COUNT = 1;
+const FILE_NAME = `mocks.json`;
+
+const TITLES = [
+  `Продам книги Стивена Кинга`,
+  `Продам новую приставку Sony Playstation 5`,
+  `Продам отличную подборку фильмов на VHS`,
+  `Куплю антиквариат`,
+  `Куплю породистого кота`,
+];
+
+const SENTENCES = [
+  `Товар в отличном состоянии.`,
+  `Пользовались бережно и только по большим праздникам.`,
+  `Продаю с болью в сердце...`,
+  `Бонусом отдам все аксессуары.`,
+  `Даю недельную гарантию.`,
+  `Если товар не понравится — верну всё до последней копейки.`,
+  `Это настоящая находка для коллекционера!`,
+  `Если найдёте дешевле — сброшу цену.`,
+  `Таких предложений больше нет!`,
+  `При покупке с меня бесплатная доставка в черте города.`,
+];
+
+const CATEGORIES = [
+  `Книги`,
+  `Разное`,
+  `Посуда`,
+  `Игры`,
+  `Животные`,
+  `Журналы`,
+];
+
+const OfferType = {
+  OFFER: `offer`,
+  SALE: `sale`,
+};
+
+
+const SumRestrict = {
+  MIN: 1000,
+  MAX: 100000,
+};
+
+const PictureRestrict = {
+  MIN: 1,
+  MAX: 16
+};
+
+const getPictureFileName = (number) => `item${number.toString().padStart(2, 0)}.jpg`;
+
+const generateOffers = (count) => (
+  Array(count).fill({}).map(() => ({
+    category: [CATEGORIES[getRandomInt(0, CATEGORIES.length - 1)]],
+    description: shuffle(SENTENCES).slice(1, 5).join(` `),
+    picture: getPictureFileName(getRandomInt(PictureRestrict.MIN, PictureRestrict.MAX)),
+    title: TITLES[getRandomInt(0, TITLES.length - 1)],
+    type: Object.keys(OfferType)[Math.floor(Math.random() * Object.keys(OfferType).length)],
+    sum: getRandomInt(SumRestrict.MIN, SumRestrict.MAX),
+  }))
+);
+
+module.exports = {
+  name: `--generate`,
+  run(args) {
+    const [count] = args;
+    const countOffer = Number.parseInt(count, 10) || DEFAULT_COUNT;
+    const content = JSON.stringify(generateOffers(countOffer));
+
+    fs.writeFile(FILE_NAME, content, (err) => {
+      if (err) {
+        return console.error(`Can't write data to file...`);
```

Comment:
Согласно пункту 4 задания:
```
В случае успешного формирования моковых данных и записи их в файл, программа должна завершаться с кодом 0. В случае ошибки с кодом 1.
```
```suggestion
        console.error(`Can't write data to file...`);
        process.exit(ExitCode.error);
```